### PR TITLE
Qingwei/trivial fixed

### PR DESCRIPTION
**Pull request by qingweibinary**

### CODE REVIEW

File: src/asset-picker/AssetPickerFilter.js [hunk 1]
```diff
@@ -19,10 +20,10 @@ export default class AssetPickerFilter extends Component {
 		actions.updateAssetPickerFilter(e);
 	}
 
-	// componentDidMount() {
-	// 	const assetSearchNode = findDOMNode(this.refs.assetSearch);
-    //     setTimeout(() => assetSearchNode.focus(), 1000);
-    // }
+    componentDidMount() {
+		const assetSearchNode = findDOMNode(this.refs.asset);
+        setTimeout(() => assetSearchNode.firstChild.focus(), 300);
```
Comment (qingweibinary) [inline src/asset-picker/AssetPickerFilter.js]: setTimeout is a hack ~ I was not able to find which part of the app got the focus, seems to be <body>, but I dunno why

Any idea?

File: public/styles.css [hunk 2]
```diff
@@ -236,6 +236,10 @@ nav.sidebar {
 }
 
 .full-screen-overlay {
+    z-index: 9998;
```
Comment (borisyankov) [inline public/styles.css]: Are you sure this is needed? It shouldn't be.
z-index is relevant only for relative and absolutely positioned things The content of full-screen-overlay are not using this, so they do not need to be z-indexed.

File: src/asset-picker/AssetPickerFilter.js [hunk 3]
```diff
@@ -32,7 +33,7 @@ export default class AssetPickerFilter extends Component {
 		return (
 			<div className="asset-picker-filter">
 				<InputGroup
-					ref="assetSearch"
+					ref="asset"
```
Comment (borisyankov) [inline src/asset-picker/AssetPickerFilter.js]: Hmmmm, why the rename, seems a bit error prone?

File: src/asset-picker/AssetPickerFilter.js (hunk 1 above)
Comment (borisyankov) [inline src/asset-picker/AssetPickerFilter.js]: Is it working with setTimeout though? It wasn't for me.

File: public/styles.css (hunk 2 above)
Comment (qingweibinary) [inline public/styles.css]: I see, I got it wrong, will fix

File: src/asset-picker/AssetPickerFilter.js (hunk 3 above)
Comment (qingweibinary) [inline src/asset-picker/AssetPickerFilter.js]: OK, will name it back

File: src/asset-picker/AssetPickerFilter.js (hunk 1 above)
Comment (qingweibinary) [inline src/asset-picker/AssetPickerFilter.js]: Because of firstChild ~

**Comment by qingweibinary** [comment]
@borisyankov fixed , test passed

**Pull request merged**